src/main/java/winstone/cmdline/Option.java

```diff
@@ -203,6 +205,39 @@ public int get(Map<String, String> args, int defaultValue) {
         }
     }
 
+    public static class ODebugInt extends OInt {
+
+        public ODebugInt(String name, int defaultValue) {
+            super(name, defaultValue);
+        }
+
+        @Override
+        public int get(Map<String, String> args) {
+            switch (super.get(args)) {
+                // before switching to java.util.Logging, winstone used a (1:9) range for log levels
+                case 1:
+                    return Level.OFF.intValue();
+                case 2:
+                    return Level.SEVERE.intValue();
+                case 3:
+                    return Level.WARNING.intValue();
+                case 4:
+                    return Level.INFO.intValue();
+                case 6:
+                    return Level.FINE.intValue();
+                case 7:
+                    return Level.FINER.intValue();
+                case 8:
+                    return Level.FINEST.intValue();
+                case 9:
+                    return Level.ALL.intValue();
+                case 5:
+                default:
+                    return Level.INFO.intValue();
+            }
+        }
+    }
+
```

Comment:
The mapping is a bit funky, but this was just extracted from existing code in Winstone and generalized to apply to both `Option` and `WinstoneControl`.